NitroxModel-Subnautica/DataStructures/DataExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using UnityEngine;
```

Comment:
Doesn't this still require NitroxModel to take a dependency on unity?  Should this reside somewhere else?

Reply:
This is NitroxModel-Subnautica which needs to take dependecy on Subnautica to provide a bridge between NitroxModel and programs that depend on Subnautica.